src/main/kotlin/app/Configurator.kt

```diff
@@ -119,8 +138,29 @@ object Configurator {
     /**
      * Gets list of repos from merger of all configuration levels.
      */
+    fun getLocalRepos(): List<LocalRepo> {
+        return config.localRepos
+    }
+
+    /**
+     * Gets list of temprorary saved repos.
+     */
     fun getRepos(): List<Repo> {
-        return config.repos
+        return repos
+    }
+
+    /**
+     * Gets profile URL from merger of all configuration levels.
+     */
+    fun getProfileUrl(): String {
+        return config.profileUrl
```

Comment:
Because client can not decide what is his url to profile page even if username is known.